src/command_table_start.go

```diff
@@ -248,6 +248,9 @@ func commandTableStart(s *Session, d *CommandData) {
 		}
 	}
 
+	// Record the initial status of the game
+	t.NotifyStatus()
```

Comment:
why is this getting moved

Reply:
It's been un-moved on the latest commit. I did separate out the NotifyStackDirections since that is needed before the cards are drawn

Reply:
Otherwise any cards on player's initial hands will have white arrows.